src/attachments/attachments.js

```diff
@@ -0,0 +1,76 @@
+const request = require("request-promise-native");
+
+const randomId = require("../utils/randomId");
+
+const addAttachment = async (
+  apiClient,
+  { taskId, commentText, attachmentUrl, attachmentFileName }
```

Comment:
I'm gonna merge this, but it would be best for a comment-making function to take an array of attachments to upload, since a comment can have multiple attachments and text (this is different from Nozbe Personal, where a file is a type of comment and has no more text)

Reply:
👍 I'll see what I can do.